tests/test_sources.py

```diff
@@ -262,3 +263,17 @@ def test_ingest_name(db):
 def test_strip_unicode_dashes(input, expected):
     result = strip_unicode_dashes(input)
     assert result == expected
+
+@pytest.mark.parametrize('input,ra, dec, expected', [
+    #2 cases whose names are NOT in the database
+    ("Apple", 144.395292,29.528028, False),
+    ("Banana", 3.9888,4.2511, False),
```

Comment:
add to the expected results any names which are also returned at that ra and dec